UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/GLTFSceneExporter.cs

```diff
@@ -637,7 +724,108 @@ private static bool IsPrimitive(GameObject gameObject)
 				&& gameObject.transform.localRotation == Quaternion.identity
 				&& gameObject.transform.localScale == Vector3.one
 				&& ContainsValidRenderer(gameObject);
+		}
+		
+		private void ExportSkin(string name, GameObject[] primitives, Node node)
+		{
+			foreach (var prim in primitives)
+			{
+				var smr = prim.GetComponent<SkinnedMeshRenderer>();
+				if (_existedSkins.TryGetValue(smr.rootBone.GetInstanceID(), out SkinId skinId))
```

Comment:
Shouldn't use the smr.rootbone's instanceid as the key, different skinnmeshrender'  may have same rootBone

Reply:
I have fixed this problem, I will commit it later. Thank you!